sdk-core/address.md

```diff
@@ -63,34 +60,14 @@ factory = AddressFactory("erd")
 
 address = factory.create_from_bech32("erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th")
 address = factory.create_from_hex("0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1")
-address = factory.create_from_pubkey(bytes.fromhex("0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1"))
-```
-
-## AddressConverter
-
-```
-class AddressConverter:
-    constructor(hrp: string = DEFAULT_HRP);
-
-    // Converts a bech32 string to a public key.
-    bech32ToPubkey(value: string): bytes;
-
-    // Converts a public key to a bech32 string.
-    pubkeyToBech32(pubkey: bytes): string;
-```
-
-Example of usage:
-
-```
-converter = AddressConverter("erd")
-
-pubkey = converter.bech32_to_pubkey("erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th")
-bech32 = converter.pubkey_to_bech32(bytes.fromhex("0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1"))
+address = factory.create_from_public_key(bytes.fromhex("0139472eff6886771a982f3083da5d421f24c29181e63888228dc81ca60d69e1"))
 ```
 
 Module-level functions:
 
 ```
 // Note that the first input parameter is received as an interface, but the return value is a concrete type [see Guideline **`in-ifaces-out-concrete-types`**].
-compute_contract_address(deployer: IAddress, deployment_nonce: INonce, address_hrp: string): Address:
+compute_contract_address(deployer: IAddress, deployment_nonce: INonce, address_hrp: string): Address;
```

Comment:
we might think of a component that will contain all these helpers. Reason: to implement all these compute functions we need some inner components (shard coordinator, blockchain hook (!) and such). To overcome these continuous sub-components we might create them once (on the constructor) and use them whenever we want.

Reply:
Added `AddressComputer`.